src/navigation/effects.js

```diff
@@ -327,6 +331,42 @@ export const emojiPreset = {
   transitionSpec: { close: closeSpec, open: emojiOpenSpec },
 };
 
+export const emojiPresetWallet = {
+  ...emojiPreset,
+  cardOverlay: ({ style }) => {
+    const backgroundOpacity = style.opacity.interpolate({
+      inputRange: [-1, 0, 0.975, 2],
+      outputRange: [0, 0, 1, 1],
+    });
+
+    return (
+      <Animated.View
+        pointerEvents="none"
```

Comment:
why it's animated?

Reply:
Will remove this.

Reply:
actually, we do need it for opacity style.